src/dmd/dclass.d

```diff
@@ -992,6 +992,23 @@ extern (C++) class ClassDeclaration : AggregateDeclaration
     {
         v.visit(this);
     }
+
+    /****************************************
+     * Gets an array of all implemented interfaces of this class. This includes
```

Comment:
What does it mean, "gets an array" ? I.e. how does the user access the getted array?